deployments/charts/kuma/templates/cp-rbac.yaml

```diff
@@ -158,37 +148,14 @@ rules:
       - kuma.io
     resources:
       - meshgatewayinstances/status
-    verbs:
-      - get
-      - patch
-      - update
-  # for the OwnerReferencesPermissionEnforcement plugin
-  - apiGroups:
-      - kuma.io
-    resources:
       - meshgatewayinstances/finalizers
+      - pods/finalizers
+      - meshes/finalizers
+      - dataplanes/finalizers
     verbs:
       - get
       - patch
       - update
-  - apiGroups:
-      - kuma.io
-    resources:
-      - externalservices
-      - faultinjections
-      - healthchecks
-      - trafficlogs
-      - traffictraces
-    verbs:
-      - get
-      - list
-      - watch
-      {{- if eq .Values.controlPlane.mode "zone" }}
-      - create
-      - update
-      - patch
-      - delete
-  {{- end }}
```

Comment:
Yeah, I think it was like this because we don't create default versions of these policies. I was thinking it would be good to have a similar RBAC for all policies regardless of Zone/Globa/Standalone. Even if it means granting more permissions to `kuma-control-plane`.